integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -63,4 +72,65 @@ public void remove_key(String key) throws IOException {
         map.remove(key);
         sessionRequestBody = objectMapper.writeValueAsString(map);
     }
+
+    @When("user sends a valid request to authorization end point")
+    public void user_sends_a_valid_request_to_authorization_end_point()
+            throws IOException, InterruptedException, URISyntaxException {
+        List<NameValuePair> params = new ArrayList<>();
```

Comment:
Remove lines 79-87
&
just pass in the redirectUri and clientId to the `sendAuthorizationRequest` method. See comment on `IpvCoreStubUtil` class